Source/WebKit/UIProcess/API/Cocoa/_WKTextExtraction.swift

```diff
@@ -54,9 +115,154 @@ extension WKTextExtractionItem {
 extension WKTextExtractionContainerItem {
     let container: WKTextExtractionContainer
 
-    init(container: WKTextExtractionContainer, rectInWebView: CGRect, children: [WKTextExtractionItem]) {
+    init(
+        container: WKTextExtractionContainer,
+        rectInWebView: CGRect,
+        children: [WKTextExtractionItem],
+        eventListeners: WKTextExtractionEventListenerTypes,
+        ariaAttributes: [String: String],
+        accessibilityRole: String,
+        nodeIdentifier: String?
+    ) {
         self.container = container
-        super.init(with: rectInWebView, children: children)
+        super
```

Comment:
does the linter complain if the `.init(` itself is the same line as the super?

Reply:
Yes